Program.cs

```diff
@@ -47,18 +47,16 @@ class Program {
             Console.CursorTop = 2; Console.CursorLeft = 0;
             Console.WriteLine ($"Solution {i} of {sFinalSoln.Count}");
             var soln = sFinalSoln[i - 1];
-            for (int j = 0; j <= 8; j++) {
-               if (j == 0)
-                  Console.WriteLine ("┌───┬───┬───┬───┬───┬───┬───┬───┐");
-               else if (j == 8) {
-                  PlaceQueens (soln[j - 1]);
-                  Console.WriteLine ("└───┴───┴───┴───┴───┴───┴───┴───┘");
-               } else {
-                  PlaceQueens (soln[j - 1]);
-                  Console.WriteLine ("├───┼───┼───┼───┼───┼───┼───┼───┤");
-               }
+            Console.WriteLine ("┌───┬───┬───┬───┬───┬───┬───┬───┐");
+            for (int j = 0; j < 7; j++) {
+               PlaceQueens (soln[j]);
+               Console.WriteLine ("├───┼───┼───┼───┼───┼───┼───┼───┤");
             }
-            Console.ReadKey ();
+            PlaceQueens (soln[7]);

```

Comment:
You could put this inside the loop as loop is mainly used for placing the queens.